README.md

```diff
@@ -1,15 +1,106 @@
+without Concourse, you will need to run a command separately to uninstall Locust.
+
+### Using Concourse
+
+```
+fly --target development execute --config [FILEPATH-FROM-YOUR-LOCAL-MACHINE]/ras-rm-concourse/tasks/helm/deploy/deploy-locust-manually.yaml \
+    --input helm=[FILEPATH-FROM-YOUR-LOCAL-MACHINE]/ras-rm-performance-tests/_infra/helm \
+    --var NAMESPACE=[TARGET-NAMESPACE] \
+    --var PROJECT=[TARGET-PROJECT]\
+    --var ENV=[TARGET-ENV] \
+    --var CLUSTER=[TARGET-CLUSTER] \
+    --var LOCUST_FILE=[LOCUST-CONFIG-FILE].py \
+    --var TEST_RESPONDENTS=8 \
+    --var LOCUST_USERS=8 \
+    --var LOCUST_SPAWN_RATE=1 \
+    --var LOCUST_RUN_TIME=60m \
+    --var USER_WAIT_TIME_MIN_SECONDS=2 \
+    --var USER_WAIT_TIME_MAX_SECONDS=3 \
+    --var TASK_SLEEP_TIME=1800 \
+    --include-ignored
+   ```
+
+### Using helm
+
+```
+helm upgrade --install locust [FILEPATH-FROM-YOUR-LOCAL-MACHINE]ras-rm-performance-tests/_infra/helm/locust \
+    --namespace [TARGET-NAMESPACE] \
+    --set-string gcp.project=[TARGET-PROJECT] \
+    --set-string env=[TARGET-ENV] \
+    --set-string namespace=[TARGET-NAMESPACE] \
+    --set-string locust.loadtest.environment.test_respondents=8 \
+    --set-string locust.loadtest.environment.user_wait_time_min_seconds=2
+    --set-string locust.loadtest.environment.user_wait_time_max_seconds=3
+    --set-string locust.master.environment.LOCUST_USERS=8 \
+    --set-string locust.master.environment.LOCUST_SPAWN_RATE=1 \
+    --set-string locust.master.environment.LOCUST_RUN_TIME="60m" \
+    --set-string locust.loadtest.locust_locustfile=[LOCUST-CONFIG-FILE].py
+```
+
+These below commands are to be used when Performance is not the target project
+
+```
+    --set-string locust.loadtest.locust_host=http://frontstage.[TARGET-NAMESPACE].svc.cluster.local:9000 \
+    --set-string locust.loadtest.environment.case=http://case.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.collection_exercise=http://collection-exercise.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.collection_instrument=http://collection-instrument.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.party=http://party.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.sample=http://sample.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.sample_file_uploader=http://sample-file-uploader.[TARGET-NAMESPACE].svc.cluster.local:8080 \
+    --set-string locust.loadtest.environment.survey=http://survey.[TARGET-NAMESPACE].svc.cluster.local:8080
+```
+
+Once completed, run:
+`helm uninstall locust --namespace [NAMESPACE] --ignore-not-found`
+
+### Database clear down between tests
+
+Firstly, whitelist you IP using the ras commands for the target environment. Then get the database credentials
```

Comment:
Make this specifically the performance environment. This should never be needed anywhere other than that.

Reply:
I've made a note that this should only be performance or dev (when wanting quick testing).

Reply:
Made more relevant.